

drivers/media/video/arv.c: In function `iic':
drivers/media/video/arv.c:147: `PLDI2CDATA' undeclared (first use in this function)
drivers/media/video/arv.c:147: (Each undeclared identifier is reported only once
drivers/media/video/arv.c:147: for each function it appears in.)
drivers/media/video/arv.c:149: `ARVCR0' undeclared (first use in this function)
drivers/media/video/arv.c:149: `ARVCR0_VDS' undeclared (first use in this function)

etcetera

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/media/video/Kconfig |    1 +
 1 files changed, 1 insertion(+)

diff -puN drivers/media/video/Kconfig~m32r-ar-camera-driver-build-fix drivers/media/video/Kconfig
--- 25/drivers/media/video/Kconfig~m32r-ar-camera-driver-build-fix	Wed Oct  6 17:52:00 2004
+++ 25-akpm/drivers/media/video/Kconfig	Wed Oct  6 17:54:18 2004
@@ -308,6 +308,7 @@ config VIDEO_OVCAMCHIP
 
 config VIDEO_M32R_AR
 	tristate "AR devices"
+	depends on M32R
 	---help---
 	  This is a video4linux driver for the Renesas AR (Artificial Retina)
 	  camera module.
_
